pkg/manifests/types.go

```diff
@@ -48,6 +48,8 @@ type ClusterMonitoringConfiguration struct {
 	TelemeterClientConfig *TelemeterClientConfig `json:"telemeterClient,omitempty"`
 	// `ThanosQuerierConfig` defines settings for the Thanos Querier component.
 	ThanosQuerierConfig *ThanosQuerierConfig `json:"thanosQuerier,omitempty"`
+	// `NodeExporterConfig` defines settings for the Node Exporter component.
+	NodeExporterConfig *NodeExporterConfig `json:"nodeExporter,omitempty"`
```

Comment:
```suggestion
	NodeExporterConfig NodeExporterConfig `json:"nodeExporter,omitempty"`
```

Reply:
Shall we replace the pointer with their respective structures for configs of other components in the future, too?

Reply:
I don't think that we need to modify the existing code (as long as it works). But for future work, it's less error-prone to use values instead of pointers (e.g. you don't have to check for nil and deal with nil-pointer panics).